src/main/java/com/joyent/manta/monitor/MantaMonitorJerseyServer.java

```diff
@@ -141,10 +220,38 @@ private void configureServer() {
             httpsConfig.addCustomizer(src);
 
             SslContextFactory sslContextFactory = new SslContextFactory();
-            sslContextFactory.setKeyStorePath(System.getenv("KEYSTORE_PATH"));
+            // Write the keystore and the truststore files in the user's current
+            // working directory. For eg: /opt/manta-monitor/.manta-monitor/keystore
+            String userDirPath = System.getProperty("user.dir");
+
+            URI keystoreURI = getURIFromString(System.getenv("KEYSTORE_PATH"));
+
+            if ("manta".equals(keystoreURI.getScheme())) {
+                // Make the keystore file available for the sslContextFactory, by
+                // reading the remote file and writing it to a file on the host file system.
+                File keystoreTargetFile = new File(userDirPath + File.separator
+                        + MANTA_MONITOR_SUBDIR + File.separator + "keystore");
```

Comment:
Using a directory prefixed with a dot doesn't make sense when you are now using a path in the current running directory.

Reply:
Please give thought to the conventions of the operating system.

Reply:
Agreed to remove the *dot* in the name of the subdir. The File.separator should adhere to the platform's file separator.